.gitignore

```diff
@@ -29,6 +29,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
+package-lock.json
```

Comment:
you don't want to ignore this... its important to ensure your builds (`npm ci`) use the same dependencies to make your builds repeatable and bugs reproducible.

Reply:
thank you, will include this.